libs/damap/src/lib/components/dmp/dmp.component.ts

```diff
@@ -65,7 +59,6 @@ export class DmpComponent implements OnInit, OnDestroy {
   costsStep: UntypedFormGroup;
 
   // Resources
-  projectsLoaded$: Observable<LoadingState>;
   projects$: Observable<Project[]>;
```

Comment:
I think this might also no longer be needed. Could you check if that is safe to remove (and update line 249 ff accordingly)?